### PR TITLE
Errors to stderr

### PR DESCRIPTION
**Pull request by gabyx**

### CODE REVIEW

File: base-template.sh [hunk 1]
```diff
@@ -138,8 +138,8 @@ execute_lfs_hook_if_appropriate() {
     if [ "$GIT_LFS_AVAILABLE" = "true" ]; then
         git lfs "$HOOK_NAME" "$@" || return 1
     elif [ "$REQUIRES_LFS_SUPPORT" = "true" ]; then
-        echo "This repository requires Git LFS, but \`git-lfs\` was not found on your PATH."
-        echo "If you no longer want to use Git LFS, remove the \`.githooks/.lfs-required\` file."
+        echo "! This repository requires Git LFS, but \`git-lfs\` was not found on your PATH." >&2
+        echo "!  If you no longer want to use Git LFS, remove the \`.githooks/.lfs-required\` file." >&2
```
Comment (rycus86) [inline base-template.sh]: As a general theme:
- please keep the `!` on the start of the first line only (no need to have all lines start with `!`)
- please don't change the indentation on lines, they should start where the previous did in most cases
Example:

```
! This is the first line of error
  This is some text explaining what to do, etc.
```

File: base-template.sh [hunk 2]
```diff
@@ -452,7 +452,7 @@ update_shared_hooks_if_appropriate() {
                 PULL_OUTPUT=$(cd "$SHARED_ROOT" && git pull 2>&1)
                 # shellcheck disable=SC2181
                 if [ $? -ne 0 ]; then
-                    echo "! Update failed, git pull output:"
+                    echo "! Update failed, git pull output:" >&2
                     echo "$PULL_OUTPUT"
```
Comment (rycus86) [inline base-template.sh]: Redirect this to `>&2` as well.

File: base-template.sh [hunk 3]
```diff
@@ -461,7 +461,7 @@ update_shared_hooks_if_appropriate() {
                 CLONE_OUTPUT=$(git clone "$SHARED_REPO" "$SHARED_ROOT" 2>&1)
                 # shellcheck disable=SC2181
                 if [ $? -ne 0 ]; then
-                    echo "! Clone failed, git clone output:"
+                    echo "! Clone failed, git clone output:" >&2
                     echo "$CLONE_OUTPUT"
```
Comment (rycus86) [inline base-template.sh]: Redirect this to `>&2` as well.

File: base-template.sh [hunk 4]
```diff
@@ -295,7 +295,7 @@ is_trusted_repo() {
 
         # shellcheck disable=SC2181
         if [ $TRUST_ALL_RESULT -ne 0 ]; then
-            echo "! This repository wants you to trust all current and future hooks without prompting"
+            echo "! This repository wants you to trust all current and future hooks without prompting" >&2
```
Comment (rycus86) [inline base-template.sh]: This is a prompt rather than a failure, please keep it on stdout.

File: base-template.sh (hunk 1 above)
Comment (gabyx) [reply base-template.sh]: Ah I wasnt sure, so now its clear, I though you wanted to distinguise further error message from each other by indenting...

Comment (gabyx) [reply base-template.sh]: So one "!" marks one error! I correct that overall

File: base-template.sh (hunk 4 above)
Comment (gabyx) [reply base-template.sh]: The prompt comes afterwards I though (printf)

**Comment by rycus86** [comment]
Thanks a lot!

**Pull request merged**